examples-experimental/potree/package.json

```diff
@@ -17,8 +17,8 @@
     "@deck.gl/layers": "^9.0.32",
     "@deck.gl/mesh-layers": "^9.0.32",
     "@deck.gl/react": "^9.0.32",
-    "@loaders.gl/potree": "4.4.0-alpha.0",
```

Comment:
Suggestion: 
- I'd probably rename this example from `potree` to `point-tile-source` with the goal of expanding it to cover COPCSource.
- And perhaps we can even make point tile sources for 3D tiles and I3S?